templatefitter/fit_model/parameter_handler.py

```diff
@@ -236,28 +235,15 @@ def finalize(self) -> None:
 
         self._is_finalized = True
 
-    def _create_floating_parameter_mask(self) -> Tuple[bool, ...]:
-        return tuple([p_info.floating for p_info in self._parameter_infos])
+    def _create_floating_parameter_mask(self) -> np.ndarray:
+        return np.array([p_info.floating for p_info in self._parameter_infos])
 
     def _create_conversion_vector(self) -> np.ndarray:
         # Is the inverse of the floating_parameter_mask converted to integers times the parameters
         # and thus yields the fixed parameters.
         conversion_vector = np.array([0 if m else 1 for m in self.floating_parameter_mask])
```

Comment:
I think here `self.floating_parameter_mask` (wrapping the private `_floating_mask` attribute) is still `None` so that might not work. I might also have misunderstood your suggestion.

Reply:
Ah, of course... sorry for the noise!

Reply:
No problem!